packages/medusa/src/services/user.ts

```diff
@@ -62,9 +63,53 @@ class UserService extends TransactionBaseService {
    * @param {Object} config - the configuration object for the query
    * @return {Promise} the result of the find operation
    */
-  async list(selector: FilterableUserProps, config = {}): Promise<User[]> {
+  async list(
+    selector: Selector<FilterableUserProps> & { q?: string } = {},
+    config: FindConfig<FilterableUserProps> = { skip: 0, take: 20 }
+  ): Promise<User[]> {
+    const [users] = await this.listAndCount(selector, config)
```

Comment:
Todo: i ve just seen the email and wanted to share that we shouldn't do that. It in fact perform one to two more queries under the hood to get the count which add up to the response time. So if we can it is better as a general approach to differentiate the two list and listAndCount method

Reply:
All good, I have changed it to not use `listAndCount` I was just trying to follow the pattern we have in most of our other services ✌️

Reply:
ahah no problem, unfortunatelly this one is not one to follow 😂